documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +105,52 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load is stored in a ConfigMap (with the same name as the KafakRebalance custom resource) as a JSON formatted string. This JSON string consists of a JSON object with keys for each broker IDs linking to a number of metrics for each broker.
+Each metric consist of three values.
+The first is the metric value before the optimization proposal is applied, the second is the expected value of the metric after the proposal is applied, and the third is the difference between the first two values (after minus before).
+
+To extract the JSON string from the ConfigMap, use the following command:
+
+[source,shell,subs=+quotes]
+----
+kubectl get configmap _my-rebalance_ -o json | jq .data
```

Comment:
We shouldn't use emphasis ( `_`) in a code example.

Reply:
Also it would be nice to provide a jq example that spits out just the load json:
```
jq '.["data"]["brokerLoad.json"]'
```

Reply:
When you do that the actual JSON is not parseable by `jq`. It contains a load of escape characters etc. This may be linked to the way we are encoding the string.

Reply:
I think we shouldn't be using `encodePrettily` is should just be `encode`

Reply:
@tomncooper,Hello, Actually  ( _)  this change was suggested by @laidan6000 , mostly due to consistency If I remember correctly

Reply:
I have changed it to encode() now

Reply:
Ok I tried this change and I think it should def be `encode`. The JSON can be parsed with `jq` using this command:
```
jq '.["data"]["brokerLoad.json"]|fromjson|.'
```
The user can then add their query onto the end of that string.

Reply:
Is still wonder if it would have been possible to encode the data directly into the `ConfigMap` builder and avoid JSON all together. Then the user could convert the config map data with `-o json` or `-o yaml`.

Reply:
Okay, Got it. I will push these changes in some minutes